frontend/src/components/ShootWorkers/GWorkerGroups.vue

```diff
@@ -0,0 +1,45 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div
+    v-if="!isShootWorkerless"
+    class="d-flex align-center"
+  >
+    <g-worker-group
+      v-for="workerGroup in shootWorkerGroups"
+      :key="workerGroup.name"
+      v-model="workerGroupTab"
+      :worker-group="workerGroup"
+      :cloud-profile-name="shootCloudProfileName"
+      :shoot-item="shootItem"
+      class="ma-1"
+    />
+  </div>
+  <span v-else>
+    Workerless Cluster
+  </span>
+</template>
+
+<script>
+
+import GWorkerGroup from '@/components/ShootWorkers/GWorkerGroup'
+
+import { shootItem } from '@/mixins/shootItem'
+
+export default {
+  name: 'GWorkerGroups',
```

Comment:
```suggestion
```
let's not set names for components, except there is a good reason to do so. We should probably also get rid of the other places where a name is set. I'm not aware of any place in our code base where it makes sense to do so

Reply:
I removed it for components involved in this PR